### ISSUE TITLE
Stop shipping with StableRNGs?

### ISSUE THREAD

**Issue by jakobnissen** [BioJulia/BioSequences.jl#187]
Right now, BioSequences ship with StableRNGs for reproducible RNG behaviour. But... why, exactly? Do most users really care about precise RNG reproducibility?

And if they do, they would need to manually set the seed of an RNG object anyway, and then they might as well just use StableRNGs themselves and pass in the RNG.

Should we just remove this dependency for 3.0?

**Comment by kescobo** [comment]
Yeah - I don't think we need it. I think sometimes, it's nice to have as a test dependency if you want to have reproducible random tests generated (though lately, I've been just building examples by hand).

**Comment by jakobnissen** [comment]
Agreed, we should keep it as a test dependency.

**Comment by BenJWard** [comment]
Yeah, let's get rid of shipping it w/ BioSequences!
